docs/modules/servers/partials/configure/smtp-hooks.adoc

```diff
@@ -371,6 +372,7 @@ Example handlerchain configuration for `smtpserver.xml`:
 	<handler class="org.apache.james.smtpserver.DKIMHook">
 		<forceCLRF>true</forceCLRF>
 		<onlyForSenderDomain>apache.org</onlyForSenderDomain>
+        <validatedEntities>envelope,headers</validatedEntities>
```

Comment:
```suggestion
                <validatedEntities>envelope,headers</validatedEntities>
```

Reply:
Why the tab? It's not inside an xml tag like `onlyForSenderDomain`?

Reply:
Should there be a TAB inside?

Reply:
Or do you mean:

```
	<onlyForSenderDomain>
                 <domain>apache.org</domain>
                 <validatedEntities>envelope,headers</validatedEntities>
        </onlyForSenderDomain>
```

? would require few more changes in the structure then

Reply:
I am not sure about Benoit idea. But I suggest in the Antora doc that `validatedEntities` have the same indent level as `onlyForSenderDomain`, as the code currently is, no?

Reply:
It's already on the same level? at least in code and from the format i see on github?

Reply:
From what I see:

![image](https://github.com/user-attachments/assets/e3ee5c14-14ce-47f4-86ef-3e39afea5bfe)

Reply:
Ok seems maybe there was but the ide was showing me the same... tried to repush it. is it fixed on your end?

Reply:
It is good now.